modules/bullet/collision_object_bullet.cpp

```diff
@@ -79,7 +79,7 @@ btTransform CollisionObjectBullet::ShapeWrapper::get_adjusted_transform() const
 }
 
 void CollisionObjectBullet::ShapeWrapper::claim_bt_shape(const btVector3 &body_scale) {
-	if (!bt_shape) {
+	if (bt_shape == nullptr) {
 		if (active) {
 			bt_shape = shape->create_bt_shape(scale * body_scale);
```

Comment:
A matter of taste, but I prefer the original `if (!bt_shape)`, or better yet, the opposite: `if (bt_shape) return;` to avoid needing to indent the code.

Reply:
The result is the same, but in this way it's much more clear that it's a pointer. The indentation just make the code more clear, in this case, for this reason I did it so.